Extension/lib/content-script/preload.js

```diff
@@ -94,13 +94,14 @@
      * Overrides window.WebSocket running the function from websocket.js
      * https://github.com/AdguardTeam/AdguardBrowserExtension/issues/203
      */
+    /*global initPageMessageListener, overrideWebSocket*/
     var initWebSocketWrapper = function () {
-        if (overrideWebSocket
-            && typeof overrideWebSocket == 'function') {
+        if (typeof overrideWebSocket == 'function') {
+            initPageMessageListener();

```

Comment:
It was global before. I just wrapped this pageMessageListener into a function to avoid unnecessary initialization.